lib/generators/transcript/install_generator.rb

```diff
@@ -11,6 +11,8 @@ def generate_model
         "actor:references{polymorphic}",
         "receiver:references{polymorphic}",
         "receiver_serialized:jsonb",
+        "meta_object:references{polymorphic}",
```

Comment:
I’m not sure I see why you would attach it as an object. Instead shouldn’t it just be a general JSON structure where you can add _any_ data you want? That way you’re not constrained to just another association — it’s truly a “metadata” field since you can add anything in any shape you want.